tools/GenerateConcordKeys.cpp

```diff
@@ -187,10 +192,20 @@ int main(int argc, char** argv) {
 
     config.cVal = (n - (3 * config.fVal) - 1) / 2;
 
+#ifdef USE_CRYPTOPP
     std::vector<std::pair<std::string, std::string>> rsaKeys;
+#elif USE_EDDSA_OPENSSL
+    std::vector<std::pair<std::string, std::string>> eddsaKeys;
+#endif
+
     for (uint16_t i = 0; i < n + ro; ++i) {
+#ifdef USE_CRYPTOPP
```

Comment:
Isn't there a way of making it so that only the key generation code is with an ifdef?

Outputting a key to a file should be generic.

Reply:
At present, only the key generation part is inside `#ifdef`.
Later, the `#ifdef`s will be removed and a factory method to select the current sign-verify algo will be implemented.

Reply:
Wasn't this behavior supposed to be a part of this feature?